plugins/uefi-capsule/fu-uefi-common.c

```diff
@@ -100,8 +100,8 @@ fu_uefi_get_built_app_path(GError **error)
 		return NULL;
 	prefix = fu_common_get_path(FU_PATH_KIND_EFIAPPDIR);
 
-	source_path = g_strdup_printf("%s/fwupd%s.efi%s", prefix, suffix, "");
-	source_path_signed = g_strdup_printf("%s/fwupd%s.efi%s", prefix, suffix, ".signed");
+	source_path = g_strdup_printf("%s/fwupd%s.efi", prefix, suffix);
+	source_path_signed = g_strdup_printf("%s/fwupd%s.efi.signed", prefix, suffix);
```

Comment:
I think you got this wrong in the fixup

Reply:
(Just use the string you just built was my point)

Reply:
aha, you mean the 2nd one, right!

Reply:
resolved